src/main/java/frc/robot/commands/autonomousCommands/AutonomousShooterLightsCommand.java

```diff
@@ -0,0 +1,42 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
Maybe you just haven't developed it yet, but what is this class for?

Reply:
Since we had shooter used during teleop as well I didn't want to add a lights command to that directly,because I was afraid it would ruin the teleop. However, I realized this command doesn't work, and am instead adding more conditions in the limelight lock which controls the leds colors.